src/client/utils/cookie.js

```diff
@@ -121,3 +127,23 @@ export function del (document, parsedCookie) {
 
     document.cookie = format(parsedCookie);
 }
+
+export function checkDomain (document, domain) {
+    domain = prepareDomain(domain);
+
+    if (publicSuffixList[domain])
```

Comment:
Add comment with link to Public Suffix List specification and link for updating data (
Was it download from https://publicsuffix.org/list/public_suffix_list.dat?)